pynta/tasks.py

```diff
@@ -468,15 +468,16 @@ def run_task(self, fw_spec):
 class MolecularTSEstimate(FiretaskBase):
     required_params = ["rxn","ts_path","slab_path","adsorbates_path","rxns_file","path","metal","facet",
                         "name_to_adjlist_dict", "gratom_to_molecule_atom_maps",
-                        "gratom_to_molecule_surface_atom_maps","opt_obj_dict",
-                                "vib_obj_dict","IRC_obj_dict","nslab","Eharmtol","Eharmfiltertol","Ntsmin","max_num_hfsp_opts"]
-    optional_params = ["out_path","spawn_jobs","nprocs",]
+                        "gratom_to_molecule_surface_atom_maps","irc_mode",
+                        "vib_obj_dict","opt_obj_dict","nslab","Eharmtol","Eharmfiltertol","Ntsmin","max_num_hfsp_opts"]
+    optional_params = ["out_path","spawn_jobs","nprocs"]
     def run_task(self, fw_spec):
         gratom_to_molecule_atom_maps = {sm: {int(k):v for k,v in d.items()} for sm,d in self["gratom_to_molecule_atom_maps"].items()}
         gratom_to_molecule_surface_atom_maps = {sm: {int(k):v for k,v in d.items()} for sm,d in self["gratom_to_molecule_surface_atom_maps"].items()}
         out_path = self["out_path"] if "out_path" in self.keys() else ts_path
         spawn_jobs = self["spawn_jobs"] if "spawn_jobs" in self.keys() else False
         nprocs = self["nprocs"] if "nprocs" in self.keys() else 1
+        IRC_obj_dict = self["IRC_obj_dict"] if "IRC_obj_dict" in self.keys() else None
```

Comment:
You try to access "IRC_obj_dict" here, but it isn't a required or optional parameter so it won't ever find it.

Reply:
IRC_obj_dict in key deleted